consensus/expected.go

```diff
@@ -257,7 +263,9 @@ func (c *Expected) runMessages(ctx context.Context, st state.Tree, vms vm.Storag
 			return nil, errors.Wrap(err, "error validating block state")
 		}
 
-		receipts, err := c.processor.ProcessBlock(ctx, cpySt, vms, blk, tsMessages[i], ancestors)
+		// wrap bls messages and combine to process bls messages first
+		msgs := append(wrapMessages(blsMessages[i]), secpMessages[i]...)
```

Comment:
note for the future: this feels like one step backwards.  We should be verifying secp signatures and unwrapping them to match BLS, not rewrapping BLS in signed message with an empty signature.  Totally fine for now though.